pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
```

Comment:
Ok, so that's what I suspected. We should definitely not mix-up the two meanings of "column" here (the second being "character"). We're in luck since "characters" has the same starting letter, so we can describe the -c option with it easily :)